packages/sublime_text.rb

```diff
@@ -3,14 +3,14 @@
 class Sublime_text < Package
   description 'A sophisticated text editor for code, markup and prose.'
   homepage 'https://www.sublimetext.com/'
-  version '3.2'
+  version '3.2.2'
```

Comment:
3.2.11?

Reply:
Might also be good to use the same version scheme as sublime merge.

Reply:
Yeah I don't know why they version differently between the tools, but this is how the website lists the versions.

Reply:
Weird.